eng/outerbuild/outerBuild.targets

```diff
@@ -0,0 +1,8 @@
+<Project>
```

Comment:
Why do we need this extra file? Can we just move this target into Directory.Build.targets?

Reply:
No, if we will include this target in Directory.Build.Targets, the outer GenerateReferenceSource will always override the actual one. we only want the outer target to override the inner one if we are cross targeting.

Reply:
I see, that makes sense. Can you remove the extra "outerbuild" directory at least? I don't think we need that.